compiler/crates/relay-compiler/src/config.rs

```diff
@@ -404,7 +407,34 @@ Example file:
             header: config_file.header,
             codegen_command: config_file.codegen_command,
             load_saved_state_file: None,
-            generate_extra_artifacts: None,
+            generate_extra_artifacts: Some(Box::new(|_, project_config, _, _, artifacts| {
+                artifacts
+                    .iter()
+                    .map(|artifact| match &artifact.content {
+                        ArtifactContent::Operation {
+                            normalization_operation,
+                            text,
+                            id_and_text_hash,
+                            ..
+                        } => {
+                            if !is_operation_preloadable(&normalization_operation) {
+                                return None;
+                            }
+
+                            Some(generate_preloadable_query_parameters_artifact(
+                                project_config,
+                                normalization_operation,
+                                text,
+                                id_and_text_hash,
+                                artifact.artifact_source_keys.clone(),
+                                artifact.source_file,
+                            ))
+                        }
+                        _ => None,
+                    })
+                    .filter_map(|artifact| artifact)
+                    .collect()
+            })),
```

Comment:
For the code organization purposes, I would move it to an individual module/function with something like `default_generate_extra_artifacts_fn`